Form/Type/ColorSelectorType.php

```diff
@@ -33,7 +33,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults(array(
+        $defaults = array(
             'choices' => Colors::getAll(),
```

Comment:
I really did my best to understand here, read it 4 times, smells bad here :smiley:  ...  Keep not understanding...

Independent of Symfony version the ``configureOptions`` will trigger the deprecated message because ``Colors::getAll()`` (``if ($isInverted)`` ``true``) is always called. I did not catch how to prevent this using Symfony 2.8+